docs/user/openstack/install_upi.md

```diff
@@ -145,14 +149,13 @@ sudo subscription-manager repos --disable=* # if not done already
 
 sudo subscription-manager repos \
   --enable=rhel-8-for-x86_64-baseos-rpms \
-  --enable=openstack-16-tools-for-rhel-8-x86_64-rpms \
-  --enable=ansible-2.9-for-rhel-8-x86_64-rpms \
+  --enable=openstack-16.2-deployment-tools-for-rhel-8-x86_64-rpms \
   --enable=rhel-8-for-x86_64-appstream-rpms
 ```
 
 Then install the packages:
 ```sh
-sudo dnf install python3-openstackclient ansible python3-openstacksdk python3-netaddr
+sudo dnf install python3-openstackclient ansible-core python3-openstacksdk python3-netaddr
```

Comment:
IIUC, based on [this comment](https://github.com/openshift/installer/pull/7346/files#diff-389d382a95698df36fa6fd80e4570ce9a6ea4911a8f9eeeac4e1409ed957bc24R40), this will install both python 3.6 and 3.8. Is this a problem?
We should have the CI image match the instruction from this page.

Reply:
It depends on which distro you're installing all this, since we now have RHEL9 ...

I'll just add comments here to explain what to do in both cases.